docs/conf.py

```diff
@@ -34,6 +34,9 @@
 autorunrecord_line_replace = [
     # trailing space removal
     (r'[ ]+$', ''),
+    # turn our "eternal" date time into symbols
+    (r'Tue Jun 18 16:13:00 2019 +0000', '🗓🕓'),
+    (r'2019-06-18 16:13', '🗓🕓'),
```

Comment:
As a side note, I would use 📅 (Calendar, [U+1F4C5](https://unicodeplus.com/U+1F4C5)) instead of 🗓 (Spiral Calendar Pad, [U+1F5D3](https://unicodeplus.com/U+1F5D3)), but that's a personal preference and likely depends on my set of browser fonts (?) -- not leaving a GitHub suggestion as this would require a one-line change to fontpkg.sty, too.